ext/web/15_performance.js

```diff
@@ -90,8 +93,17 @@ webidl.converters["DOMString or PerformanceMeasureOptions"] = (
   return webidl.converters.DOMString(V, prefix, context, opts);
 };
 
-function setTimeOrigin(origin) {
-  timeOrigin = origin;
+const hrU8 = new Uint8Array(8);
+const hr = new Uint32Array(TypedArrayPrototypeGetBuffer(hrU8));
+
+function setTimeOrigin() {
+  op_time_origin(hrU8);
+  timeOrigin = hr[0] * 1000 + hr[1] / 1e6;
+}
+
+function opNow() {
+  op_now(hrU8);
+  return (hr[0] * 1000 + hr[1] / 1e6);
```

Comment:
Ditto, consider doing this fix in a separate PR - will make it much easier to review and test in separation.